elements/lisk-p2p/src/peer/inbound.ts

```diff
@@ -86,6 +97,7 @@ export class InboundPeer extends Peer {
 			'postTransactions',
 			this._handleRawLegacyMessagePostTransactions,
 		);
+		inboundSocket.on('pong', this._handlePong);
```

Comment:
We should define a new constant for 'ping' and 'pong' events at the top of the file.